docs/reference/commandline/cli.md

```diff
@@ -38,6 +38,7 @@ the [installation](../../installation/index.md) instructions for your operating
 For easy reference, the following list of environment variables are supported
 by the `docker` command line:
 
+* `DOCKER_API_VERSION` The API version to use (e.g. `1.19`)
```

Comment:
Don't think its necessary since people would need to go out of their way to make this happen so they should already know the risks.

Besides, I wouldn't want the warning message to interfere with any expected output.